GitUI/Editor/RichTextBoxXhtmlSupportExtension.cs

```diff
@@ -1142,8 +1142,8 @@ public static string GetPlainText(this RichTextBox rtb, int from, int to)
                 rtb.SelectionLength = nEnd;
 
                 rtb.EndUpdate(oldMask);
-                rtb.HideSelection = false;
-
+                rtb.Invalidate();
+                ////rtb.HideSelection = false
```

Comment:
Please remove, if this is no longer required

Reply:
I have removed the commented out code.